### ISSUE TITLE
non containment proxys

### ISSUE THREAD

**Issue by andreasmuelder** [emfjson/emfjson-jackson#91]
First of all thanks for the great work!
If have an issue with non containment proxies.
When I have a JSON file that contains a platform resource uri to another document, for example:
"$ref" : "platform:/resource/myproject/myresource#myobject"
and I open this file with the JSONResource, all cross document references are resolved transitive instead of providing a proxy to this object. Am I missing a load option or are proxies not implemted yet?

**Comment by ghillairet** [comment]
Which version are you using?

Proxies should be created only if the reference has it's attribute resolve proxies set to false.

**Comment by andreasmuelder** [comment]
I used the lasted version from your p2 update site (0.13).
But there is an newer release 0.15. that is not provided via the update site. I will check with this new version,  the code looks promising in EObjectDeserializer line 88 there is a proxy created now:
((InternalEObject) current).eSetProxyURI(URI.createURI(jp.nextTextValue())); instead of loading the full resource content. Maybe this was missing in 0.13.

**Comment by andreasmuelder** [comment]
I found the problem with the 0.13.0 release.

All cross reference proxies are resolved in 	DefaultReferenceEntry#resolve in line 52.
target = resourceSet.getEObject(uri, true);
This is a huge performance problem for large models.
I changed it to 
target = EcoreUtil.create(reference.getEReferenceType());
((InternalEObject) target).eSetProxyURI(uri);
and it works as expected now. (lazy loading of cross references)
